prdoc/pr_7607.prdoc

```diff
@@ -0,0 +1,20 @@
+title: Add Runtime Api version to metadata
+doc:
+- audience: Runtime Dev
+  description: |-
+    The runtime API implemented version is not explicitly shown in metadata, so here we add it to improve developer experience.
+
+    This closes #7352 .
+crates:
+- name: sp-api-proc-macro
+  bump: patch
```

Comment:
This patch bump is not correct. It is a major bump because it is now using a new major version of `sp-metadata-ir`.

I opened the issue https://github.com/paritytech/polkadot-sdk/issues/8142

Reply:
The CI + release should handle this though. Looks like a bug in it